doc/source/installation.rst

```diff
@@ -2,11 +2,222 @@
 Installation
 ============
 
-At the command line::
+Using pip
+---------
+Use::
 
-    $ pip install python-redfish
+    sudo pip install python-redfish
 
-Or, if you have virtualenvwrapper installed::
+Pip will install :
+
+1. The library and all dependencies into your site-packages directory
+2. Redfish client master conf file into /etc/redfish_client.conf
+3. Data file (templates) into /usr/share/redfish-client/templates
+
+Point 2 and 3 above need root access to your system. If you don't have root
+access on your system, please follow `Using pip and virtualenv`_ section.
+
+
+Using pip and virtualenv
+------------------------
+
+1. Install virtualenv and virtualenvwrapper:
+
+ Fedora 22::
+
+    dnf install python-virtualenv.noarch python-virtualenvwrapper.noarch
+
+ Ubuntu 15.04::
+
+    apt-get install python-virtualenv virtualenvwrapper
+
+2. Source virtualenvwrapper.sh::
+
+    . /usr/bin/virtualenvwrapper.sh
+
+ or::
+
+    . /usr/share/virtualenvwrapper/virtualenvwrapper.sh
+
+3. Create a redfish virtual environement::
+
+    mkvirtualenv redfish
+
+4. Install using pip::
+
+    pip install python-virtualenv
+
+ All files are installed under your virtualenv.
+
+Using the sources
+-----------------
+
+#. Follow `get the sources <http://pythonhosted.org/python-redfish/readme.html#get-the-sources>`_ section to retrieve the sources.
+#. Install from the source using::
+
+    python setup.py install
+
+
+Using rpm package
+-----------------
+
+There is currently no oficial packages for distributions.
+However part of the sources there is a mechanism to buil rpm or deb packages for distributions.
+
+The mechanism is based on `project builder <http://www.project-builder.org/>`_ tool.
+
+#. Follow `get the sources <http://pythonhosted.org/python-redfish/readme.html#get-the-sources>`_ section to retrieve the sources.
+#. Download project builder for your distribution from ftp://ftp.project-builder.org.
+#. Clone the project to your own github account.
+#. Create a .pbrc with the following containt::
+
+    [uggla@ugglalaptop ~]$ cat .pbrc
+    pbdefdir python-redfish = $ENV{'HOME'}/workspace
+    pbconfdir python-redfish = $ENV{'HOME'}/workspace/python-redfish/pbconf
+    pbconfurl python-redfish = git+ssh://git@github.com:uggla/python-redfish.git
+    pburl python-redfish = git+ssh://git@github.com:uggla/python-redfish.git
```

Comment:
I've also fixed #14 so now instructions mentionned here could be used there as well to document better how to use pb for that.